pkg/apis/workflow/v1alpha1/generated.proto

```diff
@@ -539,6 +539,7 @@ message NodeStatus {
   optional k8s.io.apimachinery.pkg.apis.meta.v1.Time finishedAt = 11;
 
   // ResourcesDuration is indicative, but not accurate, resource duration. This is populated when the nodes completes.
+  // ALPHA
```

Comment:
is this a good way to say this is alpha?

Reply:
I don't think so. Seems out of place and that syntax should be reserved for `// DEPRECATED` in my opinion.

How about: 
```
// ResourcesDuration is indicative, but not accurate, resource duration. This is populated when the nodes completes (Alpha).